python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -130,19 +129,28 @@ def __init__(self, *args, **kwargs):
                                                                      "NO_DATA", "DETECTED_NEGATIVE"]))
 
     @pipeBase.timeMethod
-    def run(self, templateExposure, exposure, subtractedExposure, psfMatchingKernel):
+    def run(self, exposure, templateExposure, subtractedExposure, psfMatchingKernel,
+            xcen=None, ycen=None, var1=None, var2=None):
         """! Perform decorrelation of an image difference exposure.
 
         Decorrelates the diffim due to the convolution of the templateExposure with the
         A&L PSF matching kernel. Currently can accept a spatially varying matching kernel but in
         this case it simply uses a static kernel from the center of the exposure.
 
-        @param[in] templateExposure the template afwImage.Exposure used for PSF matching
         @param[in] exposure the science afwImage.Exposure used for PSF matching
+        @param[in] templateExposure the template afwImage.Exposure used for PSF matching
         @param[in] subtractedExposure the subtracted exposure produced by
         `ip_diffim.ImagePsfMatchTask.subtractExposures()`
         @param[in] psfMatchingKernel an (optionally spatially-varying) PSF matching kernel produced
         by `ip_diffim.ImagePsfMatchTask.subtractExposures()`
+        @param[in] xcen X-pixel coordinate to use for computing constant matching kernel to use
+        If `None` (default), then use the center of the image.
+        @param[in] ycen Y-pixel coordinate to use for computing constant matching kernel to use
+        If `None` (default), then use the center of the image.
+        @param[in] var1 image variance for science image
```

Comment:
If you're calling the images exposure and template, you should call the variances var_exposure and var_template (or something), or call the images var1 and var2. Having the names different like this makes them hard to keep track of.